common/Cargo.toml

```diff
@@ -38,6 +38,7 @@ serde_test = "1.0"
 serde_json = "1.0"
 rand = "0.8.5"
 itertools = "0.10"
+chainstate = { path = "../chainstate"}
 
```

Comment:
This is wrong. Storage should not depend on chainstate, because chainstate typically depends on storage.

Reply:
It is only a `dev-dependency`, so it is OK because technically tests is another crate.

Reply:
I noticed it's a `dev-dependency`, but it's just wrong. The only purpose of `dev-dependency` is to reduce compilation time and isolate test dependencies. It should not be used to circumvent the dependency tree.

Reply:
I would like to argue. 🙃 But I have checked and was unable to found how the chainstate depencency is used. Perhaps I'm missing something, but probably it can simply be removed.

Reply:
Removed